src/main/java/org/prebid/server/bidder/epsilon/EpsilonBidder.java

```diff
@@ -84,22 +93,37 @@ private BidRequest createOutgoingRequest(BidRequest bidRequest) {
         for (int i = 0; i < requestImps.size(); i++) {
             final Imp imp = requestImps.get(i);
             final ExtImpEpsilon impExt = parseImpExt(imp, i);
-            modifiedImps.add(modifyImp(imp, impExt));
+            final BigDecimal bidFloor = resolveBidFloor(bidRequest,
+                    imp.getBidfloorcur(),
+                    getBidFloor(imp.getBidfloor(), impExt.getBidfloor()));
+            modifiedImps.add(modifyImp(imp, impExt, bidFloor));
```

Comment:
Is the Imp's bidfloorcur modified elsewhere?

Reply:
It will be set in modifyImp